src/main/java/view/InputView.java

```diff
@@ -0,0 +1,35 @@
+package view;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import domain.Player;
+
+public class InputView {
+
+	private static final Scanner scanner = new Scanner(System.in);
+	private static final String YES = "y";
+	private static final String NO = "n";
+
+	private InputView() {
+	}
+
+	public static List<String> askPlayerNames() {
+		System.out.println("게임에 참여할 사람의 이름을 입력하세요.(쉼표 기준으로 분리)");
+		String input = scanner.nextLine();
+		return Stream.of(input.split(",", -1))
+			.map(String::trim)
+			.collect(Collectors.toList());
+	}
+
+	public static boolean askIfHit(final Player player) {
+		System.out.println(player.getName() + "는 한장의 카드를 더 받겠습니까?(예는 y, 아니오는 n)");
+		String input = scanner.nextLine().strip();
+		if (!input.equalsIgnoreCase(YES) && !input.equalsIgnoreCase(NO)) {
+			throw new IllegalArgumentException("y또는 n을 입력해주세요");
+		}
+		return input.equalsIgnoreCase(YES);
```

Comment:
```suggestion
		return YES.equalsIgnoreCase(input);

```
상수를 통해 비교하는게 좋습니다. 왜그런지에 대해서는 고민해보세요!